content/posts/books-i-have-read.md

```
@@ -0,0 +1,62 @@
+---
+title: "Books I Have Read"
+date: 2025-04-28
+lastmod: 2025-04-28
+publish: true
+---
+
+{{< admonition note "incomplete: This list began in late 2022" >}}
+
+{{< /admonition >}}
+## Recent
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)